### PR TITLE
Style theme

### PR DESCRIPTION
**Pull request by jde-maga**
There are a lot of stuff to still work on but the theme and current layout is really needed so pass this pls

### CODE REVIEW

File: client/components/Home/Feed.tsx [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+
+import Rating from "@material-ui/lab/Rating";
+import { useFeedStyles } from "./styles";
+
+import { clamp } from "../Search/service";
+
+const mock = [
+  {
+    updatedAt: moment().format(),
+    user: "connard",
```
Comment (Jcharloi) [inline client/components/Home/Feed.tsx]: su per

File: client/components/Layout/Filters.tsx [hunk 2]
```diff
@@ -111,14 +110,15 @@ const Filters = (): ReactElement => {
           const {
             target: { id }
           } = (e as unknown) as ClickAwayEventTarget;
+          console.log("clickaway", e);
```
Comment (Jcharloi) [inline client/components/Layout/Filters.tsx]: remove console.log

File: client/components/Home/Feed.tsx (hunk 1 above)
Comment (jde-maga) [reply client/components/Home/Feed.tsx]: i was not happy this day

File: client/components/Home/FeaturedMovie.tsx [hunk 3]
```diff
@@ -0,0 +1,92 @@
+import React, { ReactElement } from "react";
+
```
Comment (Jcharloi) [inline client/components/Home/FeaturedMovie.tsx]: remove \n

File: client/components/Layout/Filters.tsx [hunk 4]
```diff
@@ -5,33 +5,32 @@ import { useIntl } from "react-intl";
...
-import InputBase from "@material-ui/core/InputBase";
 import Rating from "@material-ui/lab/Rating";
-import SearchIcon from "@material-ui/icons/Search";
 import Slider from "@material-ui/core/Slider";
 import Typography from "@material-ui/core/Typography";
 import Paper from "@material-ui/core/Paper";
-import ClickAwayListener from "@material-ui/core/ClickAwayListener";
 
-import { Popper } from "@material-ui/core";
 import useDebounce from "../../hooks/useDebounce";
```
Comment (Jcharloi) [inline client/components/Layout/Filters.tsx]: remove \n

File: client/components/Layout/Filters.tsx [hunk 5]
```diff
@@ -5,33 +5,32 @@ import { useIntl } from "react-intl";
...
 
 import history from "../../helpers/history";
```
Comment (Jcharloi) [inline client/components/Layout/Filters.tsx]: remove \n

File: client/components/Layout/Filters.tsx [hunk 6]
```diff
@@ -5,33 +5,32 @@ import { useIntl } from "react-intl";
...
 
-import { ClickAwayEventTarget } from "../../models/models";
-
 import { useFiltersStyles } from "./styles";
```
Comment (Jcharloi) [inline client/components/Layout/Filters.tsx]: remove \n

File: client/components/Layout/Header.tsx [hunk 7]
```diff
@@ -0,0 +1,112 @@
...
+  IconButton,
+  Toolbar,
+  Menu,
+  MenuItem,
+  OutlinedInput
+} from "@material-ui/core";
+
+import SearchIcon from "@material-ui/icons/Search";
+import AccountCircle from "@material-ui/icons/AccountCircle";
+
```
Comment (Jcharloi) [inline client/components/Layout/Header.tsx]: remove \n

File: client/translations/fr.json [hunk 8]
```diff
@@ -62,5 +62,11 @@
   "layout.filters.select_category.additional_collections_video": "Videos additionnelles",
   "layout.filters.select.minrating": "Note minimale",
   "layout.filters.select.maxrating": "Note maximale",
-  "layout.filters.select.reset": "Reinitialiser les filtres"
+  "layout.filters.select.reset": "Reinitialiser les filtres",
```
Comment (Jcharloi) [inline client/translations/fr.json]: Réinitialiser*

File: client/translations/fr.json [hunk 9]
```diff
@@ -62,5 +62,11 @@
...
+  "fourOhFour.title": "404",
+  "fourOhFour.content": "Page non trouvée",
+  "fourOhFour.link_menu": "Accueil",
+  "error.title": "Oups...",
+  "error.content": "Quelque chose est cassé",
```
Comment (Jcharloi) [inline client/translations/fr.json]: xD

Comment (jde-maga) [reply client/translations/fr.json]: (ne mettez pas 0 svp)

**Pull request merged**